tests/integration.rs

```diff
@@ -0,0 +1,195 @@
+use serial_test::serial;
+use std::net::{IpAddr, Ipv4Addr, SocketAddr};
+use bevy::MinimalPlugins;
+use bevy::input::{ButtonState, Input, InputPlugin};
+use bevy::input::keyboard::KeyboardInput;
+use bevy::prelude::*;
+use bevy::utils::HashMap;
+use ggrs::{Config, SessionBuilder, PlayerType, UdpNonBlockingSocket, PlayerHandle, P2PSession};
+use bevy_ggrs::{GgrsPlugin, AdvanceFrame, PlayerInputs, Rollback, Session, LocalInputResource};
+use bytemuck::{Pod, Zeroable};
+
+#[test]
+#[serial]
+fn it_runs_advance_frame_schedule_systems() -> Result<(), Box<dyn std::error::Error>> {
+    let (player1, player2) = create_players();
+    let session1 = start_session(&player1, &player2)?;
+    let mut app1 = create_app::<GgrsConfig>(session1);
+    let session2 = start_session(&player2, &player1)?;
+    let mut app2 = create_app::<GgrsConfig>(session2);
+
+    let inputs1 = HashMap::from([
+        (player1.handle, BoxInput { inp: 0 }),
+    ]);
+    let inputs_resource = LocalInputResource::<GgrsConfig> {
+        inputs: inputs1,
+    };
+    app1.insert_resource(inputs_resource);
+    for _ in 0..50 {
+        app1.update();
+        app2.update();
+    }
+
+
+    let frame_count1 = app1.world.get_resource::<FrameCount>().unwrap();
+    let frame_count2 = app2.world.get_resource::<FrameCount>().unwrap();
+    assert!(frame_count1.frame > 0);
+    assert!(frame_count2.frame > 0);
+    assert_eq!(frame_count1.frame, frame_count2.frame);
```

Comment:
Come to think of it, this assertion is wrong. It's ok if the frame count don't match. That is expected. This test should only test if the Advance frame stage systems run.